src/modules/reducers/augmintToken.js

```diff
@@ -146,25 +146,14 @@ export const refreshAugmintToken = () => {
 
 async function getAugmintTokenInfo(augmintToken) {
     const web3 = store.getState().web3Connect.web3Instance;
-    const [
-        symbol,
-        bytes32_peggedSymbol,
-        bn_totalSupply,
-        decimals,
-        feeAccount,
-        bn_feePt,
-        bn_feeMin,
-        bn_feeMax
-    ] = await Promise.all([
+    const [symbol, bytes32_peggedSymbol, bn_totalSupply, decimals, feeAccount, transferFeeStruct] = await Promise.all([
```

Comment:
I personally prefer the vertical list of variables, but I guess it was `prettier`'d